changelogs/fragments/4728-onepassword-v2.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
```

Comment:
I think this is more than a bugfix. (Maybe even major changes?)
```suggestion
minor_changes:
```

Reply:
It's a major under the hood change for more or less a bug fix. I'll go with a minor change.